tests/main/interfaces-alsa/task.yaml

```diff
@@ -0,0 +1,94 @@
+summary: Ensure that the alsa interface works.
+
+systems: [-debian-*]
+
+details: |
+    The alsa interface allows connected plugs to access raw ALSA devices.
+
+    A snap which defines a alsa plug must be shown in the interfaces list.
+    The plug must not be autoconnected on install and, as usual, must be able to
+    be reconnected.
+
+    A snap declaring a plug on this interface must be able to access to raw ALSA
+    devices, for this test we check the low level security rules by creating
+    specific devices under /dev/snd, a cgroup if needed and the ALSA state dir,
+    exercising in each caase the read or read-write permissions that must be in
+    place.
+
+prepare: |
+    echo "Given a snap declaring a plug on the alsa interface is installed"
+    . $TESTSLIB/snaps.sh
+    install_generic_consumer alsa
```

Comment:
Thanks! I'll take a look, we try to use as much as possible test snaps with code in snapd's tree, we can easily adapt them to the evolving conditions and upload to the store as needed. Also, it is good to use the smallest test snap possible to check the features exercised by the test, in this particular case we can use the tiniest consumer possible, generic-consumer https://github.com/snapcore/snapd/tree/master/tests/lib/snaps/generic-consumer